spec/services/bulk_upload/lettings/year2022/row_parser_spec.rb

```diff
@@ -599,6 +599,28 @@
       end
     end
 
+    describe "#field_85" do
+      let(:bulk_upload) { create(:bulk_upload, :lettings, user:, needstype: "2") }
```

Comment:
out of interest does create() implicitly call FactoryBot.create or is it something else?
why do we use one over the other in different test contexts?

Reply:
I think they're doing the same thing, I'm not a 100% sure but I think we can use the shorter syntax because of this in the spec_helper `config.include FactoryBot::Syntax::Methods`. I don't think there's a reason why we would use one over the other. The inconsistency probably happened due to different preferences of different people writing the tests